### DISCUSSION TITLE
No solvation atoms options

### DISCUSSION THREAD

**Discussion by xiaojinz** [hernanchavezthielemann/GRO2LAM#50] [General]
My system contains clay particles solvated in water. When I convert my input file, I found out that there was no "Solvation Atoms" popup in the panel and there was no water molecule in the final generated input file. Is there any requirement for the gro or top file so that the solvation atoms option will show up?

**Comment by hernanchavezthielemann** [comment]
Hello @xiaojinz,

I cannot say that much without looking at your input files.
But I think you can find what you are looking in the examples provided with GRO2LAM.
Maybe, check that folder, and in there the force field parameters.

Best,
Hernan.

**Comment by hernanchavezthielemann** [comment]
btw @xiaojinz, use the @ to reference me ( as I did with you ), otherwise I wont be notified.

**Reply by xiaojinz** [reply]
Thank you very much @hernanchavezthielemann ! I will try to look in the examples and keep you updated if i have any new ideas. Thanks!!
